frontend/packages/process-editor/src/utils/processActions/processActions.test.ts

```diff
@@ -0,0 +1,63 @@
+import { getPredefinedActions, isActionAvailable, isActionRequiredForTask } from './processActions';
+
+describe('processActionsUtils', () => {
+  describe('getPredefinedActions', () => {
+    it('should return predefined actions for signing', () => {
+      const result = getPredefinedActions('signing');
+      expect(result).toEqual(['write', 'reject', 'confirm', 'sign']);
+    });
+
+    it('should return predefined actions for payment', () => {
+      const result = getPredefinedActions('payment');
+      expect(result).toEqual(['write', 'reject', 'confirm', 'pay']);
+    });
+
+    it('should return predefined actions for confirmation', () => {
+      const result = getPredefinedActions('confirmation');
+      expect(result).toEqual(['write', 'reject', 'confirm']);
+    });
+
+    it('should return predefined actions for task', () => {
+      const result = getPredefinedActions('data');
+      expect(result).toEqual(['write', 'reject', 'confirm']);
+    });
+  });
```

Comment:
Maybe we should add some whitespace between the `describe`s and `it`s? Can see it has been done some places 😄

Reply:
The Prettier configuration is set up to remove the new line between describe and it when formatting files. However, I added a new line between the it blocks to give some breathing room while reading the test. 😊

Reply:
Great, it was between the `describe` blocks, and between the `it` blocks I was thinking about initially 😄